.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,7 +11,10 @@ of the problem, prefixed by the primary affected package.
 
 
 ### What did you do?
-<!-- Please include a link to a minimal reproducer here. -->
+<!--
+Please include a link to a minimal reproducer here. Preferably this should
+be a complete runnable example at https://play.golang.org/.
```

Comment:
we should also perhaps require they display the result of `gonum.Version()`. (perhaps with a link to a pre-shared template?)

Reply:
I'm not sure how stable that will be in the current implementation. I note though that in the issue associated with the CL that added foreign imports that there is discussion about versioning. Probably it would be best to wait until that is decided.

Reply:
I was merely advocating for replacing the link to `https://play.golang.org` with that kind of link:
https://play.golang.org/p/OqLpL0_kyV5

but ok.